docs/components/content/ComponentCard.vue

```diff
@@ -186,8 +187,8 @@ const propsToSelect = computed(() => Object.keys(componentProps).map((key) => {
 const code = computed(() => {
   let code = `\`\`\`html
 <${name}`
-  for (const [key, value] of Object.entries(componentProps)) {
-    if (value === 'undefined' || value === null) {
+  for (const [key, value] of Object.entries(fullProps.value)) {
+    if (value === 'undefined' || value === null || key === 'name') {
```

Comment:
It won't work to omit `name` here as some props are named like this: https://ui-2hs33orwh-nuxt-js.vercel.app/dev/elements/icon

Reply:
So do you want to remove the `name` prop of the form elements from the `.md` file?

Or simply exclude `icon` component?

Reply:
Yes I think it's better to remove the `name` from the form element.